### PR TITLE
Feature/ci conda flux

### PR DESCRIPTION
**Pull request by RamonAra209**
- Added `flux-core` and `flux-sched` to requirements. 
- Added `flux` related tests 
- Removed `swift-t` from requirements and tests due to dependency conflict

### CODE REVIEW

File: .gitlab/llnl-ci-conda.yml [hunk 1]
```diff
@@ -138,7 +138,7 @@ conda-tests:
     - export contact="arambula2@llnl.gov"
     - export imnumber="LLNL-MI-834241"
     - python3 ci/tests/test.py -s
-    - for TEST in parsl rp swift-t; do
+    - for TEST in flux parsl rp parsl-flux rp-flux maestro; do
```
Comment (mtitov) [inline .gitlab/llnl-ci-conda.yml:141]: `maestro` is not in conda
(besides this, all looks good)

**Comment by j-woz** [comment]
What is the dependency conflict?

File: .gitlab/llnl-ci-conda.yml (hunk 1 above)
Comment (RamonAra209) [reply .gitlab/llnl-ci-conda.yml]: Done

**Comment by RamonAra209** [comment]
@j-woz 

Swift-t requires python version`3.6 >= python < 3.7`

`Flux-core` and `flux-sched` need version `>= 3.8`

**Comment by mtitov** [comment]
@j-woz is it ok to resolve it outside of this PR?

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**